### PR TITLE
feat: add a `viewBox` to the `svg` element for better scaling

### PR DESCRIPTION
**Pull request by blmage**
Thanks for publishing this package, along with [use-elapsed-time](https://github.com/vydimitrov/use-elapsed-time) it has been very helpful for my last side project!

Everything is working fine so far, except that the timer doesn't seem to scale down well - adding a `viewBox` to the `<svg>` element should be enough to help browsers figure what to do.

### CODE REVIEW

File: packages/web/src/CountdownCircleTimer.tsx [hunk 1]
```diff
@@ -17,7 +17,12 @@ const CountdownCircleTimer = (props: Props) => {
 
   return (
     <div style={getWrapperStyle(size)}>
-      <svg width={size} height={size} xmlns="http://www.w3.org/2000/svg">
+      <svg
+        viewBox={`0 0 {size} {size}`}
```
Comment (vydimitrov) [inline packages/web/src/CountdownCircleTimer.tsx:21]: Hey @blmage, good catch! That makes sense, aren't you missing the `$` before the size here?

Comment (vydimitrov) [reply packages/web/src/CountdownCircleTimer.tsx:21]: I guess we will need the same logic for the mobile package as well :)

Comment (blmage) [reply packages/web/src/CountdownCircleTimer.tsx]: Oops, you're right! Should be all good now for both packages.

**Comment by vydimitrov** [comment]
Great, I am merging this PR! Thank you @blmage.

**Pull request merged**